knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -61,23 +101,37 @@ var _ reconcile.Reconciler = &ReconcileKnativeServing{}
 type ReconcileKnativeServing struct {
 	// This client, initialized using mgr.Client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver
-	client client.Client
-	scheme *runtime.Scheme
+	client         client.Client
+	scheme         *runtime.Scheme
+	namespacedName *types.NamespacedName
```

Comment:
What do we need this for? Seems pretty much unused below.

Reply:
Per the comments above, I think this won't be needed if we enqueue the right resources.

Reply:
Done.